test/saithrift/tests/sail3.py

```diff
@@ -1477,3 +1477,192 @@ def runTest(self):
             self.client.sai_thrift_remove_router_interface(rif_port_id)
             self.client.sai_thrift_remove_virtual_router(self.vr_id)
             print "END OF TEST"
+@group('lag')
+@group('l3')
+class L3MultipleEcmpLagTest(sai_base_test.ThriftInterfaceDataPlane):
+    total_changing_ports = 15
+    first_changing_port = 2
+    total_dst_port = 16
+    v4_enabled = 1
+    v6_enabled = 1
+    ip_mask = '255.255.0.0'
+    addr_family = SAI_IP_ADDR_FAMILY_IPV4
+    lag_members = []
+    nhop_group =  0
+    lag = 0
+    lag_rif = 0
+    port_rifs = []
+    neighbors = []
+    nhops = []
+    routes = []
+    vr_id = 0
+    mac_action = SAI_PACKET_ACTION_FORWARD
+    src_port = 0
+    mac_pool = ['00:11:22:33:44:50',
+               '00:11:23:33:44:51',
+               '00:11:24:33:44:52',
+               '00:11:25:33:44:53',
+               '00:11:26:33:44:54',
+               '00:11:27:33:44:55',
+               '00:11:28:33:44:56',
+               '00:11:29:33:44:57',
+               '00:11:30:33:44:58',
+               '00:11:31:33:44:59',
+               '00:11:32:33:44:60',
+               '00:11:33:33:44:61',
+               '00:11:34:33:44:62',
+               '00:11:35:33:44:63',
+               '00:11:36:33:44:64',
+               '00:11:37:33:44:65',
+               '00:11:38:33:44:66']
+    
+    
+    def setup_rifs(self, first_rif_port, port_list):
```

Comment:
I can do it with only one var
because if I will pass lag_size,I will know the number of ports in ecmp group
ecmp_rifs = total_ports - lag_size